src/app/widgets/cat-source/cat-source-tev/cat-source-tev.component.ts

```diff
@@ -55,9 +55,28 @@ export class CatSourceTeVComponent implements OnInit {
     return false;
   }
 
+  goToMap() {
+    let formattedGlon = this.source.format(this.d.glon, false).toString();
```

Comment:
Maybe move all of this code into a property `aladin_target_string` on the source class?